sr_ethercat_hand_config/demos/demo_l.py

```diff
@@ -273,7 +273,7 @@
 # FUNCTION DEFINITIONS #
 ########################
 
-def secuence_ff():
+def secuence_ff():  # pylint: disable=R0915
```

Comment:
I might be wrong, but I guess it is supposed to be sequence and not secuence. Could we change all the iterations of this atrocity, please?